src/integrationTests/community.ts

```diff
@@ -1,408 +1,287 @@
 import { createAction } from '@reduxjs/toolkit';
```

Comment:
Why is this file still present? Does it contain only scenario for the test that was not yet converted to jest?

Reply:
This file contains all the 'stories' that jest test uses. It would be unreadable if I'd keep that in one test file. I've grouped all the assertions and actions dispatches under relevant names i.e. createCommunity